### PR TITLE
SDK-2465 support advanced identity profile in IDV

### PR DESCRIPTION
**Pull request by laurent-yoti**

### CODE REVIEW

File: src/idv_service/session/create/identity_profile/advanced/advanced.identity.profile.scheme.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   * @param {string} type
+   * @param {string} objective
+   * @param {string} label
+   * @param {AdvancedIdentityProfileSchemeConfig} [config]
+   */
+  constructor(type, objective, label, config) {
+    Validation.isString(type, 'type');
+    this.type = type;
+    Validation.isString(objective, 'objective', true);
+    if (label) this.objective = objective;
```
Comment (dhivya-pattabhiraman) [inline src/idv_service/session/create/identity_profile/advanced/advanced.identity.profile.scheme.js:17]: this line should be: if (objective) this.objective = objective;

Comment (laurent-yoti) [reply src/idv_service/session/create/identity_profile/advanced/advanced.identity.profile.scheme.js:17]: 🦅 👁️ !!!!

Thanks!

File: src/idv_service/session/retrieve/get.session.result.js [hunk 2]
```diff
@@ -94,6 +95,19 @@ class GetSessionResult {
       /** @private */
       this.identityProfile = new IdentityProfileResponse(response.identity_profile);
     }
+
+    if (response.advanced_identity_profile) {
+      Validation.isPlainObject(response.advanced_identity_profile, 'advanced_identity_profile');
+      console.log('advanced_identity_profile:');
```
Comment (dhivya-pattabhiraman) [inline src/idv_service/session/retrieve/get.session.result.js:101]: The console.log needs to be removed.

File: src/idv_service/session/retrieve/get.session.result.js [hunk 3]
```diff
@@ -94,6 +95,19 @@ class GetSessionResult {
...
+      /** @private */
+      // eslint-disable-next-line max-len
+      this.advancedIdentityProfile = new AdvancedIdentityProfileResponse(response.advanced_identity_profile);
+    }
+
+    if (response.advanced_identity_profile_preview) {
+      console.log('advanced_identity_profile_preview:');
+      console.log(JSON.stringify(response.advanced_identity_profile_preview, null, 2));
+    }
```
Comment (dhivya-pattabhiraman) [inline src/idv_service/session/retrieve/get.session.result.js:110]: Probably this whole section needs to be removed. I haven't seen advanced_identity_profile_preview in the response or maybe one to confirm with Pawel.

**Review by abdalmajeed-yoti — APPROVED** [review]

**Review by dhivya-pattabhiraman — APPROVED** [review]

**Pull request merged**